pkgs/tools/security/whatweb/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchFromGitHub, bundlerEnv, ruby }:
+
+let
+  gems = bundlerEnv {
+    name = "whatweb-env";
+    inherit ruby;
+    gemdir = ./.;
+  };
+
+in
+stdenv.mkDerivation rec {
+
+  pname = "whatweb";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
+    owner = "urbanadventurer";
+    repo = "whatweb";
+    rev = "v${version}";
+    sha256 = "sha256-HLF55x4C8n8aPO4SI0d6Z9wZe80krtUaGUFmMaYRBIE=";
+  };
+
+  prePatch = ''
+    substituteInPlace Makefile --replace "/usr/lib" "$out"
```

Comment:
```suggestion
```
`/usr/lib` is not even in the Makefile.

Reply:
It was a typo, it should be `/usr/local` which is on the Makefile

Reply:
But why is `/usr/local` used in the first place and also only used for macOS?

Reply:
On macOS it will install on that specified path. Still we won't want to use it and try with `$out` instead.